problems/5.loops.js

```diff
@@ -1,33 +1,48 @@
 /** Returns every even number from 0 to a given number
-* @param {number} num - The number to iterate up to
-* @param {number[]} - An array of even numbers from 0 to {num}
-*/
-function evenNumbers() {
+ * @param {number} num - The number to iterate up to
+ * @param {number[]} - An array of even numbers from 0 to {num}
+ */
+function evenNumbers(array) {
+    let newArray = [];
+    let arr = Math.floor(array.lenght % 2 === 0);
```

Comment:
misspelled length

Reply:
That happens to me with this things i Install on vs code trying to be fancy ended up displacing my level of comfortability.